test/integration/backward_compatible/nearcache/hazelcast_eventual_nearcache.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file ever used ? If so, for what ?

Reply:
Yes, we use this while initializing the cluster in a test

Reply:
Yeah I saw it now. I used find usages feature of jetbrains and it said no usages.